README.md

```diff
@@ -195,6 +195,39 @@ check_pipeline:
 
 Note that the `GITLAB_API_TOKEN` variable can be configured in Settings > CI/CD.
 
+## kiwicom/isort
+
+- Base image: `python:3.7-alpine`
+- Packages: [`isort`](https://github.com/timothycrosley/isort/)
+
+Image used to format python code using [`pre-commit`](https://pre-commit.com) hooks and to check if all the imports are correctly sorted on CI.
+
+`pre-commit` hook example:
+
+```yaml
+- repo: local
+  hooks:
+    - id: isort
+      name: sort-python-imports
+      language: docker_image
```

Comment:
`isort` can also be run in `pre-commit` as a plain python package, like [here](https://github.com/kiwicom/kiwi-platform-py/blob/master/.pre-commit-config.yaml#L19), but sure, I understand providing this info now that we'll have the docker image.